src/model/model.js

```diff
@@ -0,0 +1,68 @@
+import { EventEmitter } from "events";
+
+class Model extends EventEmitter{
+    constructor(){
+        super();
+        this.state = {
+            questions: [{
+                questionid: 0,
+                userid: 0,
+                title: "O intrebare legitima",
+                text: "Un text de intrebare legitima",
+                creationdate: "azi",
+                tags:["Java", "JDBC"],
+                score:0 
+            },{
+                questionid: 1,
+                userid: 0,
+                title: "O intrebare legitima 2",
+                text: "Un text de intrebare legitima 2",
+                creationdate: "candva",
+                tags: ["C#"],
+                score:0
+            }],
+            newQuestion:{
+                questionid: -1,
+                userid: "",
+                title: "",
+                text: "",
+                creationdate: "",
+                tags: [],
+                score:-1
+            },
+            loggedUserId: 0
+        }
+    }
+
+    addQuestion(questionid, title, text, creationdate, tags, score){
```

Comment:
It would be better to determine the ID of the new question inside the model. In assignment 3, we will get the new ID from the backend.